learning-to-learn/learning-to-learn.md

```diff
@@ -91,6 +91,6 @@ Participants write a self-reflective journal entry covering:
 
 ### Supplemental Resources
 
-- [Growth Mindset (video)](https://www.youtube.com/watch?v=EyIF5VUOJc0) - This is summary of a book Growth Mindset by Carol Dweck, which depends on her research how do kids tackle with difficult problems
-- [Learning to Learn by Sarah Drasner](https://css-tricks.com/learning-to-learn/) - It contains some usefull tips which will help you to learn
-- [The Power of Believing You Can Improve (video)](https://www.youtube.com/watch?v=_X0mgOOSpLU) - A TEDx video by Carol Dweck in which she explain how mindset can affect activity of brain 
+- [Growth Mindset (video)](https://www.youtube.com/watch?v=EyIF5VUOJc0) - This is summary of a book Growth Mindset by Carol Dweck, which depends on her research how do kids tackle with difficult problems.
+- [Learning to Learn by Sarah Drasner](https://css-tricks.com/learning-to-learn/) - It contains some usefull tips which will help you to learn.
```

Comment:
usefull -> useful

Reply:
@Kundan28 I made the change. Please review my PR